my-dsl.md

```diff
@@ -2,37 +2,85 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+[Lout](http://www.adrianjwells.freeuk.com/lout.pdf) is a high-level markup language, similar to LaTex.
 
 # Domain
 _Describe the language's domain in five words._
 
+Typesetting documents, diagrams, and illustrations.
 
 # Computational model
 _What is the underlying computational model of this language? To answer this 
 question, provide a high-level description (no more than 100 words) of the 
 computation that occurs when someone executes a program in this language._
 
+Lout is a document formatter, so it reads a line of the Lout document and evaluates the expression(s) to generate a line of formatted text to add to the output document. It uses lazy evaluation to handle expressions, and can format computer program code in multiple languages, including C++, Java, Python, Perl, Ruby and Haskell.
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
+If document formatting is considered to be a tight enough domain, then Lout is very domain-specific, as it was designed solely for this purpose and has practically no use outside of it.
 
 # Internal or external?
 _Is the language implemented as an internal or external DSL? 
 Justify your answer._
 
+This language is mostly external. It draws from Tex in some ways, but overall has its own grammar.
+
+Here is a sample of code from the [Wikipedia page](https://en.wikipedia.org/wiki/Lout_(software)):
+
+```
+# This is a comment.
+
+# Use the `doc' document class and its default style.
+@SysInclude { doc }
+
+@Document
+@InitialFont { Times Base 10p }
+//
+
+# Beginning of document contents.
+@Text @Begin
+
+@PP
+This is a paragraph.  One can easily embed @B { bold } or
+@I { italic } text.  One can also easily change the style of
+text, such as { Helvetica Base } @Font { changing the font
+being used }.
+
+@BeginSections
+@Section @Title { The First Section }
+@Begin
+
+@PP
+This is the content of a section.
+
+@End @Section
+@EndSections
+
+@End @Text
+# End of the document.
+```
 
 # Host language
 _What language(s) was (were) used to implement the DSL?_
 
+Lout was first developed with influence from the eqp equation formatter and the Scribe system, and since then the only non-original extensions and improvements were the optimal paragraph breaking and automatic hyphenation of Tex.
```

Comment:
Still unsure what language were used to implement the DSL. Unless eqp equation formatter and the Scribe system are the languages in question. Don't know enough about them to be sure.